woocommerce/woocommerce
# Improve hero product chessboard pattern spacing

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:








### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new page or post.
2. Insert the `Hero Product Chessboard` pattern.
3. Check the titles on the bottom-left square are bold and an H5.
4. Check the margin between the two blocks (title+paragraph) is 50px and the margin between title and paragraph is 25px.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Improve hero product chessboard pattern spacing
#### Comment 

</details>
